lintrunner_adapters/adapters/django_upgrade_linter.py

```diff
@@ -0,0 +1,127 @@
+"""Upgrade your Django project code."""
+
+from __future__ import annotations
+
+import argparse
+import concurrent.futures
+import logging
+import os
+import sys
+from typing import Tuple, cast
+
+from django_upgrade.data import Settings
+from django_upgrade.main import apply_fixers
```

Comment:
Is it possible to run the command line directly so we don't need to import the module here?

Reply:
using run_command?

Reply:
So, when its preferable to write 'native' adapters?

Reply:
I don't have a strong opinion, but it feels cleaner to not have to import from the module (apis may change since I assume they are intended for commandline use), and not have to install the packages in the CI for testing (I blame doctest - great if there was a way to ignore import errors in doctest)